openhcl/underhill_core/src/servicing.rs

```diff
@@ -79,6 +79,9 @@ mod state {
         /// Shared pool information.
         #[mesh(10001)]
         pub shared_pool_state: Option<page_pool_alloc::save_restore::PagePoolState>,
+        /// Private pool information.
+        #[mesh(10002)]
```

Comment:
Shouldn't it replace shared_pool_state or we indeed want both?

Reply:
We want both - there will be instances in the future where both will exist (CVMs with TDISP devices)